multiversx_sdk_cli/tests/test_cli_staking_provider.py

```diff
@@ -57,7 +56,6 @@ def test_create_new_delegation_contract_with_provided_gas_limit(capsys: Any):
             str(alice),
             "--nonce",
             "7",
-            "--estimate-gas",
```

Comment:
We should document this breaking change in the PR description (so that we don't forget about it when writing the release notes).

Reply:
Added a short description.